lunatrace/bsl/docker-compose.yaml

```diff
@@ -107,12 +107,22 @@ services:
     extra_hosts:
       - "host.docker.internal:host-gateway"
 
-#  datadog:
-#    image: datadog/agent:latest
-#    environment:
-#     DD_LOGS_ENABLED: true
-#     DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL: true
-#     DD_CONTAINER_EXCLUDE_LOGS: "name:datadog-agent"
+  datadog:
+    image: datadog/agent:latest
+    environment:
+     DD_LOGS_ENABLED: "true"
+     DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL: "true"
+     DD_CONTAINER_EXCLUDE_LOGS: "name:datadog-agent"
+     DD_APM_ENABLED: "true"
```

Comment:
do we want apm on dev? this may leak other stuff from dev environments because it gets data from the host.